roles/ceph-defaults/defaults/main.yml

```diff
@@ -657,6 +657,8 @@ dashboard_rgw_api_user_id: ceph-dashboard
 dashboard_rgw_api_admin_resource: ''
 dashboard_rgw_api_no_ssl_verify: False
 dashboard_frontend_vip: ''
+prometheus_frontend_vip: ''
+alertmanager_frontend_vip: ''
```

Comment:
@fmount shouldn't those variables default to "{{ dashboard_frontend_vip }}" ?

Reply:
Nope, this assumption could be wrong and it depends on the specific infra and how the three components are exposed.
For instance, in the OSP context, prometheus and alertmanager have frontend endpoints (VIP) on the storage network, while grafana and the ceph dashboard are exposed on a composable network that can be reach by the operator (it's routed).